# request
---
# response
geometry_msgs/PointStamped graspPoint